assets/css/styles.css

```diff
@@ -6,31 +6,31 @@
   --font2: Open Sans, sans-serif;
   --logo-height: clamp(50px, 3em, 500px);
   --side-padding: 2rem;
-  --background-color: #ffffff;
-  --accent-color: #f4f7ff;
+  --background-color-high-contrast: #fffaff;
+  --background-color-low-contrast: #f4f5f4;
+  --accent-color-low-contrast: #f05d5e;
+  --accent-color-high-contrast: #fdecec;
   --background-image-offset: 75%;
-  --text-color: #000000;
-  --button-padding-top-bottom: clamp(2px, 0.25em, 10px);
-  --button-padding-sides: clamp(8px, 1em, 15px);
-  --radius-general: 0.4em;
+  --text-color: #303036;
+  --radius-general: 0.25em;
   --paragraph-width: calc(var(--background-image-offset) - 10%);
   --bold-font-weight: 350;
   --scroll-progress-height: 0.5rem;
   --ideal-max-width: 60rem;
   --shadow1: rgba(0, 0, 0, 0.15);
-  --shadow2: rgba(0, 0, 0, 0.13);
+  --button-font-size: 0.75em;

```

Comment:
I would rename this to `--font-size-small` or something similar, since it's not just used on buttons

Reply:
done